### PR TITLE
[bitnami/minio]: fix missing `metrics.enabled` docs.

### LINKED ISSUE #29218: [bitnami/minio]: `metrics.prometheusRule` also requires `metrics.enabled` which doesn't exist.
### Name and Version

bitnami/minio 14.6.7

### What steps will reproduce the bug?

the docs for `metrics.prometheusRule` mention `metrics.enabled` but `metrics.enabled` isn't documented in `values.yaml`. Weirdly, `metrics.serviceMonitor` doesn't require `metrics.enabled`.

### PR DESCRIPTION
**Pull request by Karitham**
### Description of the change

Add documentation for `metrics.enabled`, currently required by `metrics.prometheusRule.enabled`

### Applicable issues

- fixes #29218

### Additional information

I didn't want to have to bump major by adding `metrics.enabled` to `metrics.serviceMonitor` checks, so I just added a todo comment. Let me know if instead we should add it and bump major.

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/minio/templates/servicemonitor.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Copyright Broadcom, Inc. All Rights Reserved.
 SPDX-License-Identifier: APACHE-2.0
 */}}
-
+{{- /* TODO: On major version upgrade, add .Values.metrics.enabled to that check */}}
```
Comment (juan131) [inline bitnami/minio/templates/servicemonitor.yaml:5]: Given the default values is `false`, there shouldn't be any problem on adding the parameter on a minor version? I mean, there shouldn't be any upgradability issue, right?

File: bitnami/minio/values.yaml [hunk 2]
```diff
@@ -1188,6 +1188,8 @@ metrics:
   ## To allow public access without authentication for prometheus metrics set environment as follows.
   ##
   prometheusAuthType: public
+  ## @param metrics.enabled Enable metrics
+  enabled: false
```
Comment (juan131) [inline bitnami/minio/values.yaml:1192]: For consistency with other charts:
```suggestion
  ## @param metrics.enabled Enable the export of Prometheus metrics
  ##
  enabled: false
```

File: bitnami/minio/templates/servicemonitor.yaml (hunk 1 above)
Comment (Karitham) [reply bitnami/minio/templates/servicemonitor.yaml:5]: The people that are running with `metrics.serviceMonitor.enabled=true` but don't use the `metrics.prometheusRule` should have had no reason (at least no documented reason) to set `metrics.enabled=true`, so we would be removing the service monitor. This sounds like a breaking change to me but maybe I see it the wrong way

Comment (juan131) [reply bitnami/minio/templates/servicemonitor.yaml:5]: Fair enough! Let's keep the TODO then.

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**